identity_data/src/commonMain/kotlin/net/thechance/mena/identity/data/dto/auth/mappers/LoginReponseDtoMapper.kt

```diff
@@ -0,0 +1,11 @@
+package net.thechance.mena.identity.data.dto.auth.mappers
+
+import net.thechance.mena.identity.data.dto.auth.LoginResponseDto
+import net.thechance.mena.identity.domain.entity.AuthToken
+
+fun LoginResponseDto.toDomain(): AuthToken{
```

Comment:
```suggestion
fun LoginResponseDto.toDomain(): AuthToken{
```
fun LoginResponseDto.toEntity(): AuthToken{